Automattic/newspack-blocks
# feat: add Portuguese translation to the blocks

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a (partial) Portuguese translation to the blocks. 

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Navigate to WP Admin > Settings > General, and switch the site language to Portuguese.
3. Add a homepage posts block and donate block to the editor; confirm that at least the strings that appear on the front-end (both in the editor and on the actual front-end) are translated. 

### Other information:
